node-operators/networks/full-node.md

```diff
@@ -293,10 +289,6 @@ The next step is to create the systemd configuration file. Note that you have to
          --ws-port {{ networks.parachain.ws }} \
          --pruning=archive \
          --state-cache-size 1 \
-         --unsafe-rpc-external \
-         --unsafe-ws-external \
-         --rpc-methods=Safe \
-         --rpc-cors all \
          --log rpc=info \
```

Comment:
This isn't a security concern, but why have collators log rpc stuff?

Reply:
fixed